composer.json

```diff
@@ -30,5 +30,10 @@
     "autoload": {
         "psr-0": { "Liip\\TranslationBundle": "" }
     },
-    "target-dir": "Liip/TranslationBundle"
+    "target-dir": "Liip/TranslationBundle",
```

Comment:
you should switch to PSR-4 rather than PSR-0 + target-dir. target-dir is considered a legacy setting in Composer